launcher/src/test/java/org/apache/spark/launcher/LauncherServerSuite.java

```diff
@@ -152,6 +175,102 @@ public void testTimeout() throws Exception {
     }
   }
 
+  @Test
+  public void testSparkSubmitVmShutsDown() throws Exception {
+	LauncherServer launchServer = (LauncherServer)Mockito.mock(LauncherServer.class);
```

Comment:
style nits:

- code is cleaner if you just static import `Mockito.*`
- space after casts: `(LauncherServer) mock(...)`